pkg/tsdb/sqleng/sql_engine.go

```diff
@@ -849,7 +853,7 @@ func convertSQLTimeColumnToEpochMS(frame *data.Frame, timeIndex int) error {
 
 	origin := frame.Fields[timeIndex]
 	valueType := origin.Type()
-	if valueType == data.FieldTypeTime || valueType == data.FieldTypeNullableTime {
+	if valueType == data.FieldTypeTime || valueType == data.FieldTypeNullableTime || valueType == data.FieldTypeNullableString {
```

Comment:
Why do we need this here? Is it for scenarios where a 'time' column is represented as a string date? If so, should we also add `data.FieldTypeString` there?

Reply:
I should have commented on this. This was needed because of the bump of go-sql-driver/mysql. They changed how the unsigned types returns.